x/marker/spec/03_messages.md

```diff
@@ -358,6 +358,22 @@ This service message is expected to fail if:
 - Attributes cannot be normalized
 - Marker denom cannot be found or is not a restricted marker
 
+## Msg/UpdateSendDenyListRequest
+
+UpdateSendDenyList allows signers that have admin authority or via gov proposal to add and remove addresses to the deny send list for a restricted marker.
+
++++ https://github.com/provenance-io/provenance/blob/373d0ee8faeaa9e3b70d658e6069ab1781e6ce5e/proto/provenance/marker/v1/tx.proto #L295-L308
+
++++ https://github.com/provenance-io/provenance/blob/373d0ee8faeaa9e3b70d658e6069ab1781e6ce5e/proto/provenance/marker/v1/tx.proto #L310-L311
+
+This service message is expected to fail if:
+
+- Remove list has an address that does not exist in current deny list
+- Add list has an attribute that already exist in current deny list
+- Invalid address format in add/remove lists
+- Marker denom cannot be found or is not a restricted marker
+- Signer is does not have admin authority or is not from gov proposal
```

Comment:
I think you meant to say `Signer does not have admin authority or is not from gov proposal`